projects/packages/publicize/src/social-image-generator/class-setup.php

```diff
@@ -23,6 +23,22 @@ public function init() {
 		add_action( 'wp_after_insert_post', array( $this, 'set_meta' ), 10, 2 );
 	}
 
+	/**
+	 * Get the parameters for the token body.
+	 *
+	 * @param string $text Text to use in the generated image.
+	 * @param string $image_url Image to use in the generated image.
+	 * @param string $template Template to use in the generated image.
+	 * @return array
+	 */
+	public function get_token_body( $text, $image_url, $template ) {
+		return array(
+			'text'      => $text,
+			'image_url' => $image_url,
+			'template'  => $template,
+		);
+	}
```

Comment:
This is extracted into a separate function so that we can run a test that ensures we're passing in all required keys.